src/api/app/controllers/webui/package_controller.rb

```diff
@@ -390,8 +393,9 @@ def set_file_details
       end
     rescue ActiveXML::Transport::Error => e
       # TODO crudest hack ever!
-      if e.summary == 'service in progress'
+      if e.summary == 'service in progress' && @expand == 1
         @expand = 0
+        @service_running = true
```

Comment:
@mschnitzer JFYI the return doesn't matter here as I assign an instance variable which makes it also accessible in ``show``